basic/src/test/java/org/jboss/hal/testsuite/test/configuration/webservices/WebServicesTestCaseAbstract.java

```diff
@@ -40,23 +41,6 @@
     @Drone
     protected WebDriver browser;
 
-    protected void navigateToClientConfigurationHandlerChainTab(String clientConfigurationName, String handlerChainTabLabel) {
-        page.navigate();
-        page.switchToClientConfiguration();
-        page.getResourceManager()
-                .selectByName(clientConfigurationName)
-                .view();
-        page.switchSubTab(handlerChainTabLabel);
-    }
-
-    protected void navigateToEndpointConfigurationHandlerChainTab(String endpointConfigurationName, String handlerChainTabLabel) {
-        page.navigate();
-        page.switchToEndpointConfiguration();
-        page.getResourceManager()
-                .selectByName(endpointConfigurationName)
-                .view();
-        page.switchSubTab(handlerChainTabLabel);
-    }
 
     protected void createHandlerChain(String handlerChainName, String handlerChainProtocolBindings) {
```

Comment:
Methods with similar name (`create*`) are used across testsuite to create resources in model. Please change its name to make it clear, that this method creates resource in GUI.